frontend/components/menu/MenuSearchResult.vue

```diff
@@ -3,7 +3,7 @@
     <button
       @click="showTooltip = !showTooltip"
       @keydown.shift.tab="onShiftTab"
-      class="relative flex items-center justify-center style-cta rounded-full w-6 h-6 elem-shadow-sm"
+      class="relative flex items-center justify-center style-cta rounded-full w-6 h-6 elem-shadow-sm three-dot-button"
```

Comment:
Right here if you change `three-dot-button` to `max-sm:absolute max-sm:pt-4 max-sm:pr-4` it should copy the functionality of your custom class with normal Tailwind.